apis/python/src/tiledbsoma/types.py

```diff
@@ -11,10 +11,12 @@
     NPInteger = np.integer[npt.NBitBase]
     NPFloating = np.floating[npt.NBitBase]
     NDArray = npt.NDArray[np.number[npt.NBitBase]]
```

Comment:
While we're here this needs to be `NPNDArray`

We already have `SparseNDArray` and `DenseNDArray` for SOMA, and this really needs the `NP` prefix like the other NumPy types already have, and as the Pandas types have the `PD` prefix

Reply:
Makes sense; renamed it [here](https://github.com/single-cell-data/TileDB-SOMA/pull/772/commits/cd21d417d4c2bdc42f3263c959dfa2a1735446d4) :+1: